docs/client/chain.md

```diff
@@ -0,0 +1,22 @@
+# Lean Consensus Experimental Chain
+
+## Configuration
+
+### Time parameters
+
+| Name                                  | Value                     |     Unit     |   Duration    |
+| ------------------------------------- | ------------------------- | :----------: | :-----------: |
+| `SLOT_DURATION_MS`                    | `uint64(4000)`            | milliseconds | 4 seconds     |
+| `PROPOSER_REORG_CUTOFF_BPS`           | `uint64(2500)`            |      bps     | 25% into slot |
```

Comment:
Since we're doing just simple `/4` interval, I'll replace these 4 params with `INTERVALS_PER_SLOT: 4`